bedrock/firefox/templates/firefox/installer-help-b.html

```diff
@@ -0,0 +1,112 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "/firefox/base/base-protocol.html" %}
+
+{# "noindex" pages should not have the canonical or hreflang tags: bug 1442331 #}
+{% block canonical_urls %}<meta name="robots" content="noindex,follow">{% endblock %}
+
+{% block page_title %}{{ _('Your download was interrupted') }}{% endblock %}
+{% block body_id %}installer-help{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('installer_help_b') }}
+{% endblock %}
+
+{% block site_header %}
+  {% with hide_nav_cta=True %}
+    {% include 'includes/protocol/navigation/menu-firefox/index.html' %}
+  {% endwith %}
+{% endblock %}
+
+{% if installer_channel == 'beta' %}
+  {% set channel_logo = 'protocol/img/logos/firefox/browser/beta/logo-word-hor-sm.png' %}
+{% elif installer_channel == 'alpha' %}
+  {% set channel_logo = 'protocol/img/logos/firefox/browser/developer/logo-word-hor-sm.png' %}
+{% elif installer_channel == 'nightly' %}
+  {% set channel_logo = 'protocol/img/logos/firefox/browser/nightly/logo-word-hor-sm.png' %}
+{% else %}
+  {% set channel_logo = 'protocol/img/logos/firefox/browser/logo-word-hor-sm.png' %}
+{% endif %}
+
+{% block content %}
+<main class="mzp-l-content">
+  <section class="mzp-c-hero">
+    <div class="mzp-c-hero-body">
+      <h1 class="mzp-c-hero-title">{{ _('Hmm... your download was interrupted') }}</h1>
```

Comment:
Nit: should this be a an ellipsis character (`…`) instead of three periods? (`...`)